library/kani_core/src/arbitrary.rs

```diff
@@ -34,7 +34,7 @@ macro_rules! generate_arbitrary {
                         unsafe { crate::kani::any_raw_internal::<Self>() }
                     }
                     fn any_array<const MAX_ARRAY_LENGTH: usize>() -> [Self; MAX_ARRAY_LENGTH] {
-                        unsafe { crate::kani::any_raw_internal::<[Self; MAX_ARRAY_LENGTH]>() }
+                        unsafe { crate::kani::any_raw_aray::<[Self; MAX_ARRAY_LENGTH]>() }
```

Comment:
Should this be:
```suggestion
                        unsafe { crate::kani::any_raw_array::<Self; MAX_ARRAY_LENGTH>() }
```
?